flake/packages.nix

```
@@ -7,15 +7,22 @@
 {
 
   perSystem =
-    { pkgs, system, ... }:
     {
+      pkgs,
+      system,
+      config,
+      ...
+    }:
+    {
+      # We want docs, palette-generator and testbeds to be checked when running
+      # `nix flake check` or `stylix-check`
```

Avoid listing the packages here, as this will likely become out-dated. Instead, something similar to https://github.com/danth/stylix/pull/1285/files#r2093694474 would be better:

```suggestion
      # Build all packages with 'nix flake check'.
```

---

There's often confusion about whether `nix flake check` will build `packages`, because it does _evaluate_ them to check they are derivations.

Maybe:
```suggestion
      # Build all packages with 'nix flake check'. By default, it only checks they are derivations.
```
?

---

What about:

```suggestion
      # Build all packages with 'nix flake check' instead of only verifying they
      # are derivations.
```